src/core_atmosphere/Registry.xml

```diff
@@ -880,6 +880,9 @@
 			<var name="h_oml_initial"/>
 			<var name="hu_oml"/>
 			<var name="hv_oml"/>
+			<var name="OPPpres"/>
+			<var name="OPPdata"/>
+			<var name="OPPcoeffs"/>
```

Comment:
Let's take a note: this addition segment is within `<stream name="restart" ` only

Reply:
@ldfowler58 is working to move those variables into the 'static' stream. @metdyn Now this work should not need much your assistance for a while. You may focus on other things you have.

Reply:
Thanks!